### ISSUE TITLE
Attacks on Mummy Shambler (Season 1) are borked/not entered correctly

### ISSUE THREAD

**Issue by cswendrowski** [foundryvtt/pf2e#1529]
In GitLab by @meneltamar on Jun 22, 2021, 12:55

**What goes wrong:**

The Jaws attack is half entered as a passive and assigned to the Fist as an "Additional Attack Effect". The Jaws attack should be a separate 1-action attack with the prerequisite of a previous grab.

The Fist attack doesn't have the proper Grab "Additional Attack Effect". Instead the Grab effect is part of the incorrect entry above.

**How to reproduce it:**

Import Mummy Shambler from the Season 1 compendium and look at its attacks and passives.
![image](http://gitlab.com/hooking/foundry-vtt---pathfinder-2e/uploads/d719386a858a3e6701aba8736c8548b9/image.png)

**How to fix:**

The Jaws attack should be a 1-action attack with an attack bonus of +7 and 1d8+3 piercing damage.
The description of the Jaws attack should be the following:

```
Requirement The mummy has a creature @Compendium[pf2e.conditionitems.kWc1fhmv9LBiTuei]{Grabbed} or @Compendium[pf2e.conditionitems.VcDeM8A5oI6VqhbM]{Restrained}


Effect The mummy makes a jaws unarmed melee Strike against the grabbed or restrained creature with an attack modifier of +7 that deals 1d8+3 piercing damage.
```

The Fist attack should have "Grab" as its "Additional Attack Effect".
